modules/logging-release-notes-5-8-0.adoc

```diff
@@ -0,0 +1,56 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-0_{context}"]
+= Logging 5.8.0
+This release includes link:https://access.redhat.com/errata/RHBA-2023:6139[OpenShift Logging Bug Fix Release 5.8.0] and link:https://access.redhat.com/errata/RHBA-2023:6134[OpenShift Logging Bug Fix Release 5.8.0 Kibana].
+
+[id="logging-release-notes-5-8-0-enhancements"]
+== Enhancements
+
+[id="logging-release-notes-5-8-0-log-collection"]
+=== Log Collection
+* With this update, you can deploy multiple, isolated, and RBAC-protected `ClusterLogForwarder` custom resource (CR) instances in any namespace. This allows independent groups to forward desired logs to any destination and also separates logging groups to interfere with another configuration. (link:https://issues.redhat.com/browse/LOG-1343[LOG-1343])
```

Comment:
This allows independent groups to forward logs to any destination while isolating their configuration from other collector deployments

Reply:
Following up from slack... we could add something akin to what @abrennan89 added for upgrade which is basically:

if you want this feature enabled in namespaces outside of "openshift-logging", you must redeploy the CLO.

Reply:
https://github.com/openshift/openshift-docs/pull/66749/files#diff-b64c3b494ef8d8bd86630f2b657d4f8967f8b1a43ed456a880f1472b35556ef4R25 for reference

Reply:
Fixed this. @abrennan89 I have also added the "Important" note in the RNs from your reference.